src/frontend/screens/Settings/sections/GamesSettings/index.tsx

```diff
@@ -154,7 +154,7 @@ export default function GamesSettings({ useDetails = true }: Props) {
         <PreferedLanguage />
       </Collapsible>
 
-      {hasCloudSaves && (
+      {hasCloudSaves && !isLinuxNative && (
```

Comment:
Fixes Cloud Save Syncing being displayed for GOG Linux games because of missing `!isLinuxNative` check